### PR TITLE
RK-9870 - ETL Controller Documentation Rewrite

### PR DESCRIPTION
**Pull request by lirook**
ETL Controller:
- Added an "ETL Controller" section in the docs side panel.
- Created an introduction page
- Created a page for k8s and docker installations
- Created a configuration page for helm and docker
- Created a new diagram for Controller connections
- Removed Linux installation method

Misc.:
- Removed "Integrations" page as it has become obsolete

### CODE REVIEW

File: docs/etl-controller-config.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+id: etl-controller-config
+title: ETL Controller Configuration
+sidebar_label: Configuration
+---
+
+---
+
+*Please note that the ETL Controller feature is only available on the Enterprise plan.*
+
+*Enterprise users - please [contact us](https://www.rookout.com/company/contact) to enable the ETL Controller feature for your organization.*
```
Comment (LiranLast) [inline docs/etl-controller-config.md:11]: Let's merge the copy of both sentences?

File: docs/etl-controller-config.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+---
+
+## Helm Values
+
+Configure the controller [k8s installation](etl-controller-k8s) by setting values in the `values.yaml` file or using `--set` in the `helm install` command.
```
Comment (LiranLast) [inline docs/etl-controller-config.md:17]: Differences between Helm 2 and 3?

File: docs/etl-controller-config.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+sidebar_label: Configuration
+---
+
+---
+
+*Please note that the ETL Controller feature is only available on the Enterprise plan.*
+
+*Enterprise users - please [contact us](https://www.rookout.com/company/contact) to enable the ETL Controller feature for your organization.*
+
+---
```
Comment (LiranLast) [inline docs/etl-controller-config.md:13]: Can we start with a quick example of what a simple case would look like?

File: docs/etl-controller-docker.md [hunk 4]
```diff
@@ -1,79 +1,69 @@
...
-## Introduction
+---
 
-The Rookout ETL Controller is a component that can be installed within your network.  
-The ETL Controller is only meant for advanced deployments meeting complex network and security requirements.  
-Using the Rookout ETL Controller you may perform all related data processing inside your own network.
+*Please note that the ETL Controller feature is only available on the Enterprise plan.*
 
-## Run as Container
+*Enterprise users - please [contact us](https://www.rookout.com/company/contact) to enable the ETL Controller feature for your organization.*
```
Comment (LiranLast) [inline docs/etl-controller-docker.md:11]: Same comment

File: docs/etl-controller-docker.md [hunk 5]
```diff
@@ -1,79 +1,69 @@
...
 
-The easiest way to deploy the ETL Controller is as a Docker container [available here](https://hub.docker.com/r/rookout/controller/).  
-The ETL Controller can be configured to your needs using environment variables. 
+---
 
-### 1. Docker
+## Installation
 
-To run as a Docker container, simply execute:
+To install the Controller as a Docker container (using [this image](https://hub.docker.com/r/rookout/controller/)), execute:
```
Comment (LiranLast) [inline docs/etl-controller-docker.md:17]: Why install? It's how to run :)

File: docs/etl-controller-docker.md [hunk 6]
```diff
@@ -1,79 +1,69 @@
...
-
-```bash
-helm repo add rookout https://helm-charts.rookout.com
-helm repo update
-helm install --name my-release rookout/controller --set controller.token=[Your Rookout Token]
-```
 <div class="rookout-org-info"></div>
 
-## Run as Daemon
+## 3. Linux Daemon
```
Comment (LiranLast) [inline docs/etl-controller-docker.md:25]: We can deprecate this feature...

File: docs/etl-controller-intro.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+id: etl-controller-intro
+title: ETL Controller
+sidebar_label: Introduction
+---
+
+---
+
+*Please note that the ETL Controller feature is only available on the Enterprise plan.*
+
+*Enterprise users - please [contact us](https://www.rookout.com/company/contact) to enable the ETL Controller feature for your organization.*
```
Comment (LiranLast) [inline docs/etl-controller-intro.md:11]: Merge sentences :)

File: docs/etl-controller-linux.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
+---
```
Comment (LiranLast) [inline docs/etl-controller-linux.md:1]: Let's remove this page

File: docs/etl-controller-config.md (hunk 2 above)
Comment (lirook) [reply docs/etl-controller-config.md:17]: --set is available in both versions

File: docs/etl-controller-docker.md (hunk 6 above)
Comment (lirook) [reply docs/etl-controller-docker.md]: Removed Linux stuff from this page

File: docs/etl-controller-linux.md (hunk 8 above)
Comment (lirook) [reply docs/etl-controller-linux.md:1]: I already removed this page from the sidebar

File: docs/etl-controller-docker.md (hunk 5 above)
Comment (lirook) [reply docs/etl-controller-docker.md:15]: It's "install" throughout all the docs for uniformity, so I think it's best to use "install" here.
If you still think I should change it, tell me :)

File: docs/etl-controller-config.md (hunk 3 above)
Comment (lirook) [reply docs/etl-controller-config.md:11]: Do you mean like a sample configuration?

Comment (LiranLast) [reply docs/etl-controller-config.md:11]: Yes

Comment (lirook) [reply docs/etl-controller-config.md:11]: There is an install example with sample configuration on each install page (docker / kubernetes) @LiranLast

**Review by LiranLast — APPROVED** [review]

**Pull request merged**